content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,140 @@
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey" style="width:100%" >}}
+
+You are able to filter the User Journey by **View**, **Action**, **Error**, and **Frustration Signal** by selecting **Events**. 
+
+{{< img src="/getting_started/session_replay/views-actions-errors.png" alt="Panel that allows for filtering by view, action, error, and frustration signal" style="width:100%" >}}
+
+While hovering over a particular time or view in the user journey, selecting **Details** opens a side panel where you can examine specific Core Web Vitals of the page, as well as other pertinent information, without leaving the replay.
+
+{{< img src="/getting_started/session_replay/details-panel.png" alt="Panel with additional details">}}
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Troubleshoot using Developer Tools
+
+Session Replay's [Browser Developer Tools][8] enable you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console" style="width:100%" >}}
+
+You can view any **Resources** associated with a view by selecting the Resources tab.
+
+{{< img src="/getting_started/session_replay/resources.png" alt="Resources associated with the view" style="width:100%" >}}
+
+**Traces** associated with a view can be found in the [Traces tab](#tracing-with-the-apm-integration).
+
+{{< img src="/getting_started/session_replay/traces.png" alt="Traces associated with the view" style="width:100%" >}}
+
+Session metadata is populated in the **Attributes** tab.
+
+{{< img src="/getting_started/session_replay/attributes.png" alt="Attributes tab of the Dev tools console" style="width:100%" >}}
+
+## Pivot to correlated data
+
+Session Replay integrates with your application's metrics, traces, and logs to give you helpful context for debugging issues. Utilizing the APM and Error Tracking integrations alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
+
+### Tracing with the APM integration
+
+Combining Session Replay with [APM traces][9] enables you to receive end-to-end visibility across frontend and backend issues, and see how code and infrastructure are impacting your user experience.
+
+Within the Traces tab, select **View Trace in APM** to see more detailed information, including errors and logs associated with the trace. 
+
+{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information" style="width:100%" >}}
```

Comment:
I think if you provide a little more specifics of what a person might find in a trace, as I commented on in lines 80-84, you can get rid of this section entirely.